spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -1699,5 +1699,18 @@ def test_kernel_restart_after_manual_restart_and_crash(ipyconsole, qtbot):
             'status'] == 'ready')
 
 
+def test_stderr_poll(ipyconsole, qtbot):
```

Comment:
Please mark this test as `flaky`, just to avoid a failure in our test suite in case the kernel takes too much time to start.